src/test/scala/it/pagopa/interop/agreementprocess/AgreementSuspensionSpec.scala

```diff
@@ -35,13 +38,14 @@ class AgreementSuspensionSpec extends AnyWordSpecLike with SpecHelper with Scala
         verifiedAttributes = agreement.verifiedAttributes,
         suspendedByConsumer = Some(true),
         suspendedByProducer = None,
-        suspendedByPlatform = Some(false)
+        suspendedByPlatform = Some(false),
+        stamps = SpecData.suspendedAgreement.stamps
       )
 
       mockAgreementRetrieve(agreement)
       mockEServiceRetrieve(eService.id, eService)
       mockTenantRetrieve(consumer.id, consumer)
-      mockAgreementUpdate(agreement.id, expectedSeed, agreement)
+      mockAgreementUpdate(agreement.id, expectedSeed, SpecData.suspendedAgreement)
```

Comment:
⚠️ this helper function generates a new agreement with new ids, it could cause some headache in future changes.
I suggest something like 
```suggestion
      mockAgreementUpdate(agreement.id, expectedSeed, agreement.copy(stamps = SpecData.suspendedAgreement.stamps))
```